### PR TITLE
Add function to get restaurant given a city and state

### LINKED ISSUE #15: Add routes and models for restaurants
- [X] Get a specific restaurant's details
- [ ] Get a set of restaurants based on filters
- [ ] Get a list of restaurants based on location

### PR DESCRIPTION
**Pull request by WinsonChen114**

### CODE REVIEW

**Review by LifeBringer — CHANGES_REQUESTED** [review]
Comment out console.log statements

File: routes/restaurant.js [hunk 1]
```diff
@@ -8,12 +8,9 @@ router.get('/', async (req, res) => {
...
-// router.get('/location/:location', async (req, res, next))
-
 // Find restaurant info provided an ID
-//First checks databse to see if restaurant is already in database,
-//if it is not, makes a call to api and stores the data
+//First checks database to see if restaurant is already in database,
+//If it is not, makes a call to api and stores the data
 router.get('/:id', async (req, res, next) => {
     try {
         console.log("Getting restaurant by id")
```
Comment (Rebecca-CYLi) [inline routes/restaurant.js]: please remove comments

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
it looks like there's a merge conflict, please resolve it and remove comments

**Review by Rebecca-CYLi — APPROVED** [review]
comments removed, G2G

**Comment by LifeBringer** [comment]
No no, don't remove comments.

**Review by LifeBringer — APPROVED** [review]
LGTM

**Pull request merged**